#165228868 deploy nodejs app with docker on aws

ucheg6 opened the pull request:
**What does this PR do**
- deploy Nodejs app with docker on Aws

**What are the tasks to be completed**
- setup docker container and image to run app locally
- setup docker container to run app on Aws
- setup Postgresql to work with docker locally and on Aws


**How can this be manually tested**
- Navigate to this IP ```13.59.134.95``` in your browser.
- Clone the repo using `git clone https://github.com/hims-team/hims-backend.git`
- make sure you have Docker installed in your local machine
- Run ```make start```
- Navigate to ```http://localhost:9000``` in your browser you should have the app live

**Any background context**
None

What are the relevant pivotal tracker stories?
[#165228868](https://www.pivotaltracker.com/story/show/165228868)
**Screenshot**

TheoOkafor commented:
<img width="587" alt="Screenshot 2019-05-07 at 12 38 41 PM" src="https://user-images.githubusercontent.com/31534129/57296518-15e97380-70c5-11e9-9e69-f9311d4ac8e3.png">
I tried running `docker build -t hims-backend` multiple times but kept getting this response in the screenshot, any idea why?

ucheg6 commented:
> <img alt="Screenshot 2019-05-07 at 12 38 41 PM" width="587" src="https://user-images.githubusercontent.com/31534129/57296518-15e97380-70c5-11e9-9e69-f9311d4ac8e3.png">
> 
> I tried running `docker build -t hims-backend` multiple times but kept getting this response in the screenshot, any idea why?

Hello @TheoOkafor Please add the last  dot '.' to the command

ucheg6 committed:
feat(aws-docker): deploy nodejs app with docker on aws

- setup project with nodejs server and postgres
- setup docker container and upload docker image on aws
- setup aws and deploy app

[Finishes #165228868]
